src/components/card/CardSidebarTabAttachments.vue

```diff
@@ -172,19 +135,6 @@ export default {
 		background-position: 10px center;
 	}
 
-	.modal__content {
-		width: 25vw;
-		min-width: 250px;
-		height: 120px;
-		text-align: center;
-		margin: 20px 20px 60px 20px;
-	}
-
-	.modal__content button {
-		float: right;
-		margin: 40px 3px 3px 0;
-	}
-
 	.attachment-list {
```

Comment:
This can also be removed.

Reply:
Are you sure? There is still an attachment-list entry: 
https://github.com/nextcloud/deck/blob/a7473e9ec9bca51c435a36bfd4255e17e5291f22/src/components/card/CardSidebarTabAttachments.vue#L33

Reply:
Ah, I thought that would already use the new AttachmentList component. Then OK, we can also migrate that later on, maybe you can add a TODO comment for that here.